make xpi doesn't work(vimperator & muttator)

##### Issue type:
- Bug report


##### Version:

```
latest master branch
```

##### Description:
make xpi produce errors like below

Building XPI...
Including en-US documentation
Including ja documentation
Replacing UUID, VERSION, DATE and UPDATEURL tags
sed: 1: "../downloads/muttator-1 ...": invalid command code .
sed: 1: "../downloads/muttator-1 ...": invalid command code .
sed: 1: "../downloads/muttator-u ...": invalid command code .
make: *** [xpi] Error 1

##### Expected behavior:
make produce xpi file


##### Steps to reproduce:

make xpi

--- a/common/Makefile
+++ b/common/Makefile
@@ -93,7 +93,7 @@ xpi:
 
 	# Remove existing locale entries from the manifest file
 	for locale in $(SUPPORTED_LOCALES); do \
-		sed -i -e "/$$locale/d" $(XPI_PATH)/chrome.manifest; \
+		sed -i'' -e "/$$locale/d" $(XPI_PATH)/chrome.manifest; \
 	done
 
 	# Package up locale specific documentation files
@@ -116,21 +116,21 @@ xpi:
 	done
 
 	# Update locale paths in manifest file
-	sed -i -e 's# ./locale/# locale/#' $(XPI_PATH)/chrome.manifest
-	sed -i -e 's#../common/#common/#' $(XPI_PATH)/chrome.manifest
+	sed -i'' -e 's# ./locale/# locale/#' $(XPI_PATH)/chrome.manifest
+	sed -i'' -e 's#../common/#common/#' $(XPI_PATH)/chrome.manifest
 
 	@echo "Replacing UUID, VERSION, DATE and UPDATEURL tags"
 	for file in `grep -rl -e "###UUID###" -e "###VERSION###" -e "###DATE###" $(XPI_PATH)` $(UPD_RDF_PATH); do \
-		sed -i -e "s,###UUID###,$(UUID),g" $${file}; \
-		sed -i -e "s,###VERSION###,$(VERSION),g" $${file}; \
-		sed -i -e "s,###MINVERSION###,$(MIN_VERSION),g" $${file}; \
-		sed -i -e "s,###MAXVERSION###,$(MAX_VERSION),g" $${file}; \
-		sed -i -e "s,###DATE###,$(BUILD_DATE),g" $${file}; \
-		sed -i -e "s,###UPDATEXPI###,$(UPDATE_URL_XPI),g" $${file}; \
+		sed -i'' -e "s,###UUID###,$(UUID),g" $${file}; \
+		sed -i'' -e "s,###VERSION###,$(VERSION),g" $${file}; \
+		sed -i'' -e "s,###MINVERSION###,$(MIN_VERSION),g" $${file}; \
+		sed -i'' -e "s,###MAXVERSION###,$(MAX_VERSION),g" $${file}; \
+		sed -i'' -e "s,###DATE###,$(BUILD_DATE),g" $${file}; \
+		sed -i'' -e "s,###UPDATEXPI###,$(UPDATE_URL_XPI),g" $${file}; \
 		if [ -n "$(UPDATE_URL)" ]; then \
-		    sed -i -e "s,###UPDATEURL###,$(UPDATE_URL),g" $${file}; \
+		    sed -i'' -e "s,###UPDATEURL###,$(UPDATE_URL),g" $${file}; \
 		else \
-		    sed -i -e "/###UPDATEURL###/d" $${file}; \
+		    sed -i'' -e "/###UPDATEURL###/d" $${file}; \
 		fi; \
 	done
 

--- a/common/Makefile
+++ b/common/Makefile
@@ -130,7 +130,7 @@ xpi:
 		if [ -n "$(UPDATE_URL)" ]; then \
 		    sed -i -e "s,###UPDATEURL###,$(UPDATE_URL),g" $${file}; \
 		else \
-		    sed -i "/###UPDATEURL###/d" $${file}; \
+		    sed -i -e "/###UPDATEURL###/d" $${file}; \
 		fi; \
 	done
 

